jablib/src/main/java/org/jabref/model/entry/field/StandardField.java

```diff
@@ -138,6 +138,7 @@ public enum StandardField implements Field {
     OWNER("owner"),
     TIMESTAMP("timestamp", FieldProperty.DATE),
     CREATIONDATE("creationdate", FieldProperty.DATE),
+    CITATIONCOUNT("Citation count"),
```

Comment:
Move at the end - the two fields before and the one after is related to date - this one not.

Reply:
Please show the excerpt here. 

I think, you ened to adapt `jablib/src/test/resources/org/jabref/logic/exporter/OldOpenOfficeCalcExportFormatContentSingleEntry.xml` accordingly. Which is OK - you add a thing there (and we have the discussion on that there. Maybe, this is really required)

(And please let us resolve the conversations so that we save one click while reviewing) - otherwise, I need to open all resolved conversations)